docker-compose.yml

```diff
@@ -20,6 +20,7 @@ keycloak:
   volumes:
   - ./.docker-compose/keycloak.env:/etc/secrets/keycloak.env
   - ./govuk/:/opt/jboss/keycloak/themes/govuk/
+  - ./govuk-social-provider-login/:/opt/jboss/keycloak/themes/govuk-social-provider-login/
```

Comment:
Just to bike-shed the name, is either `govuk-social-provider-login-only` or `govuk-social-provider-only` marginally more descriptive?